### DISCUSSION TITLE
Failure of Autograder

### DISCUSSION THREAD

**Discussion by hui0518** [prosyslab-classroom/cs424-program-reasoning#460] [Q&A] [answered]
### Your Name

Seokhui Han (한석휘)

### Question

https://www.gradescope.com/courses/840493/assignments/4937517/submissions/294099715
I submitted my repo several times, but the autograder keeps failing.
I tried several test cases on my local docker container, but there was no problem.
I also checked that there is no print statement.
Could you give me information about the failure of the autograder.
Thank you.

**Comment by spearo2** [comment, marked as answer]
확인해본결과 기존 학생분의 문제와 동일한 문제로 파악되었습니다.
https://github.com/prosyslab-classroom/cs424-program-reasoning/discussions/452

감사합니다.

---

Your problem was found to be identical with the previous question:
https://github.com/prosyslab-classroom/cs424-program-reasoning/discussions/452

Thank you

**Reply by hui0518** [reply]
답변 감사드립니다!
